packages/config/config/devices/0x0131/ne-nas-pd07u1.json

```diff
@@ -0,0 +1,161 @@
+{
+	"manufacturer": "Zipato",
+	"manufacturerId": "0x0131",
+	"label": "NE-NAS-PD07U1",
```

Comment:
```suggestion
	"label": "AH-NAS-PD07U1",
```
probably?

Reply:
Also rename the file if we do this.

Reply:
ok, How is NE or AH determined? I just copied from existing examples

Reply:
I don't know where the other label came from tbh. I just found the Alloy website where some of their older devices are listed with the device label that just looks like a prefixed version of the Shenzhen Neo variants.